test/benchmarks/driver_bench/src/driver.mts

```diff
@@ -7,6 +7,20 @@ import process from 'node:process';
 const __dirname = import.meta.dirname;
 const require = module.createRequire(__dirname);
 
+// Special tag that marks a benchmark as a spec-required benchmark
+export const SPEC_TAG = 'spec-benchmark';
+// Special tag that enables our perf monitoring tooling to create alerts when regressions in this
+// benchmark's performance are detected
+export const ALERT_TAG = 'alerting-benchmark';
+// Tag marking a benchmark as being related to cursor performance
+export const CURSOR_TAG = 'cursor-benchmark';
+// Tag marking a benchmark as being related to read performance
+export const READ_TAG = 'read-benchmark';
+// Tag marking a benchmark as being related to write performance
+export const WRITE_TAG = 'write-benchmark';
```

Comment:
Completely optional, but individual consts has made some of the imports kinda noisy, what about enum style:

```ts
const TAG = Object.freeze({
	spec: 'spec-benchmark',
	alert: 'alerting-benchmark',
} as const)
```

```js
import { TAG } from '../driver.mjs';

const tags = [TAG.alert, TAG.spec]; 
```